### PR TITLE
[ISSUE-553] Update CRDs apiVersion

### PR DESCRIPTION
**Pull request by rkiselenko**
### Issue dell/csi-baremetal#553

* Generate CRDs
* Update related CRDs

**Dont edit CRDs manually**, use `controller-gen`.

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [x] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

### CODE REVIEW

**Review by yurkov-anton — APPROVED** [review]

**Comment by safronovD** [comment]
We have newer controller-runtime version `v0.7.2` in Operator. Is it not pass for controller-gen 0.5.0?

**Review by safronovD — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -87,6 +96,7 @@ docker-push:
 
 # find or download controller-gen
 # download controller-gen if necessary
+# TODO install controller-tool >= 0.5.0 implement version freezy in go.mod
```
Comment (safronovD) [inline Makefile:99]: Please, create Github issue for TODO and add a link here

**Comment by rkiselenko** [comment]
> We have newer controller-runtime version `v0.7.2` in Operator. Is it not pass for controller-gen 0.5.0?

[`controller-runtime`](https://github.com/kubernetes-sigs/controller-runtime) is not related to [`controller-tools`](https://github.com/kubernetes-sigs/controller-tools)

File: Makefile [hunk 2]
```diff
@@ -70,8 +74,13 @@ vet:
 
 # Generate code
 generate:
-	controller-gen object paths=api/v1/deployment_types.go paths=api/v1/groupversion_info.go output:dir=api/v1
-	controller-gen crd:trivialVersions=true paths=api/v1/deployment_types.go paths=api/v1/groupversion_info.go output:crd:dir=config/crd
+	$(CONTROLLER_GEN_BIN) $(CRD_OPTIONS) paths=$(CSI_BAREMETAL_DRIVER_DIR)/api/v1/availablecapacitycrd/availablecapacity_types.go paths=$(CSI_BAREMETAL_DRIVER_DIR)/api/v1/availablecapacitycrd/groupversion_info.go output:crd:dir=$(CSI_CHART_CRDS_PATH)
```
Comment (mishoyama) [inline Makefile:77]: I prefer to generate CRDs in `csi-baremetal` repo. Here you can create `copy` target

File: Makefile [hunk 3]
```diff
@@ -5,7 +5,11 @@ IMG ?= ${REGISTRY}/csi-baremetal-operator:${TAG}
 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true"
-
+# Variables for CRDs generation, require csi-baremetal repository
+CSI_BAREMETAL_DRIVER_DIR=../csi-baremetal
+CSI_CHART_CRDS_PATH=charts/csi-baremetal-operator/crds
+# TODO install controller-tool >= 0.5.0 implement version freezy in go.mod
+CONTROLLER_GEN_BIN=./bin/controller-gen
```
Comment (mishoyama) [inline Makefile:12]: Bin to `.gitignore`?

**Review by mishoyama — APPROVED** [review]

File: Makefile (hunk 3 above)
Comment (rkiselenko) [reply Makefile:12]: Plan to add a separate target for build `controller-gen` to bin folder.

**Pull request merged**